scripts/check-diff-line-width.sh

```diff
@@ -26,6 +26,7 @@ ERROR_THRESHOLD=100
 git --no-pager diff $BASE_HASH -U0 -- cdc pkg cmd \
 	-- ':(exclude)*_gen.go' \
 	-- ':(exclude)*_gen_test.go' \
+	-- ':(exclude)*_mock.go' \
 	-- ':(exclude)*.pb.go' |
 	-- ':(exclude)*_mock.pb.go' |
```

Comment:
```suggestion
```

It looks like a mistake, there is no such in code base, please help remove this line.